### PR TITLE
Workflows docs + mesosphere quickstart

### PR DESCRIPTION
**Pull request by Tonnu**
includes #401

### CODE REVIEW

File: content/quickstarts/deployment/mesosphere.md [hunk 1]
```diff
@@ -1,392 +1,272 @@
...
-```no-highlight
-git commit -am 'added deploy pipeline'
-git push origin master
-```
+#### Creating the Workflow
 
-Once the build pipeline completes, it will have deployed to DockerHub, so we're
-ready to deploy it to Marathon next. As before, hit the **deploy to** button
-and select `marathon`.
+Now that we’ve defined all the pipelines we can chain them together! Navigate  to the **Workflows **tab, and you should see your `test pipeline` in the pipeline editor, which represents the starting point for our Workflow. Now you can add the remaining pipelines to create Workflow we want. The end result should look like this:
```
Comment (davegri) [inline content/quickstarts/deployment/mesosphere.md]: pipeline editor > workflow editor

**Comment by Tonnu** [comment]
Included #401 in this /cc @mies

**Comment by mies** [comment]
nice, are you moving marathon to quickstarts/workflows as well?

**Comment by Tonnu** [comment]
Yes that's in here

File: content/docs/pipelines/managing.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+## Managing pipelines
+
+This section describes how pipelines can be defined and updated on the hosted Wercker platform.
+
+### Creating pipelines
+
+To create a new pipeline so that it can be used in [Workflows](/docs/workflows/index.html) you must:
+
+1. Give it a name e.g.: “**deploy-to-dockerhub**”;
+2. Decide how this workflow gets executed; This can be a Git or another Workflow hook;
```
Comment (davegri) [inline content/docs/pipelines/managing.md]: workflow > pipeline
Workflow > pipeline

File: content/docs/pipelines/managing.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+1. Public
+2. View runs
+3. Execute & Manage pipelines
+4. Admin
+
+**Public**: If both your application and pipeline are set to **public** everyone will be able to see the runs executed for that pipeline. If your application is not public, all your collaborators (regardless of their permissions) will be able to see executions for that pipeline.
+
+**View runs:** Allow only collaborators with ‘view runs’ permissions to manage this pipeline
+
+**Execute & Manage pipelines**: Allow only collaborators with ‘View runs + Manage workflows’ to manage this pipeline
```
Comment (davegri) [inline content/docs/pipelines/managing.md]: View runs + Manage workflows > Execute & Manage pipelines

**Comment by mies** [comment]
lgtm

**Pull request merged**